MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Commit in .NET MAUI DataForm control | Syncfusion
```

Comment:
title: Data handling in .NET MAUI DataForm control

Reply:
Changed title and description